### DISCUSSION TITLE
Cold or heat tolerance

### DISCUSSION THREAD

**Discussion by asdkjasdfla** [jmcollin78/versatile_thermostat#300] [General]
How can I adjust tolerance in temperature before the switch turns heater on or off

**Comment by jmcollin78** [comment]
Hello, witth VTherm, there is no need of tolerance. The algorithm adapts itself at each cycle.

**Reply by asdkjasdfla** [reply]
Does the boiler need to have VTherm support? I am controlling a Shelly switch on a dumb (non-smart) boiler. I see the versatile thermostat switching on and off the Shelly switch at low tolerances of 0.2-0.3. 

Would VTherm still work if my boiler doesn't support it and I am only controlling shelly switch through versatile thermostat

**Reply by jmcollin78** [reply]
The boiler is not yet controled by VTherm. Some adds an automation to turns on the boiler if one of the VTherm is on.

**Reply by HACS-bank** [reply]
On my system, VTherm controls the zone valves, but the boiler is controlled by a physical relay box which runs the boiler if any zone requires it for any reason. 

I don't think that I am ready to let software control the boiler, until I understand all the possible failure modes. Perhaps I am too cautious - after all, they flew to the moon on software and nobody died.

**Reply by asdkjasdfla** [reply]
How do you control the valves? I mean can you share your configuration and setup. I am trying to setup zoned heating in homeassistant. There are TRVs installed on radiators in zones and a Shelly switch controls a central gas heating boiler. Currently I am using Shelly switch via versatile thermostat and the smart TRVs turn on based on schedule. However apart from a single zone, whose temperature sensor feeds the versatile thermostat, being able to call for heat no other zone can call for heat. Wondering how can I make it work
